imapclient/livetest.py

```diff
@@ -234,6 +234,17 @@ def assertNoneOrTuple(val):
         self.assertEqual(ns.other, ns[1])
         self.assertEqual(ns.shared, ns[2])
 
+    def test_select_and_unselect(self):
+        if not self.client.has_capability('UNSELECT'):
+            return self.skipTest("Server doesn't support UNSELECT")
+
+        resp = self.client.select_folder(self.base_folder)
+        self.assertEqual(resp[b'EXISTS'], 0)
+        self.client.search(['ALL'])
+        resp = self.client.select_folder(None)
```

Comment:
Obviously, yes, You got me: I did not even re-run the test after removing the `None` option...